### PR TITLE
Add training document for how to make edits

### PR DESCRIPTION
**Pull request by ncksllvn**
This PR adds a document describing how to edit content pages. This really just lays the foundation. I suspect we'll have to make plenty of changes as we go.

#### Tickets
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15555
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13768
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13767

### CODE REVIEW

File: docs/how-to-edit-content.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+### Perform your page edits
+Now that you've created your own branch, you can make all of your edits.
+
+#### 1. First, locate the page to be edited.
+
+For example, to edit `www.va.gov/disability/eligibility/`, we would navigate to [`/pages/disability/eligibility.md`](https://github.com/department-of-veterans-affairs/vagov-content/blob/awesome-new-branch/pages/disability/eligibility.md).
+
+#### 2. Click the pencil icon in the top-right corner of the panel containing the page contents to open an editor.
+
+![Click the pencil icon to open an editor](images/create-branch.png)
```
Comment (jbalboni) [inline docs/how-to-edit-content.md]: I think this is the wrong screenshot

File: docs/how-to-edit-content.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+An editor like the one below should open -
+
+![Page editor](images/page-editor.png)
+
+__Pages are written using Markdown, whi
```
Comment (jbalboni) [inline docs/how-to-edit-content.md]: Is there missing content here?

File: docs/how-to-edit-content.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+#### 4. Click the "Create pull request" button
+
+![Create pull request](images/create-pr.png)
+
+#### 5. Optionally, edit the title/description of your pull request and click "Create pull request"
+
+![Edit pull request title](images/edit-pr-title.png)
+
+Your pull request is created!
```
Comment (jbalboni) [inline docs/how-to-edit-content.md]: Is it worth mentioning the Heroku instances that people can use to view their change?

File: docs/how-to-edit-content.md (hunk 2 above)
Comment (ncksllvn) [reply docs/how-to-edit-content.md]: Ah whoops, meant to remove that!

**Pull request merged**